src/Core/Components/Pagination/PaginationState.cs

```diff
@@ -17,6 +17,11 @@ public class PaginationState
     /// </summary>
     public int CurrentPageIndex { get; private set; }
 
+    /// <summary>
+    /// Gets the current 1-based page index. The value cannot be set manually.
+    /// </summary>
+    public int DisplayedCurrentPageIndex => CurrentPageIndex + 1;
```

Comment:
I thought it was perhaps not very clear that the indices (current and last) are 0 based and it makes the template a little cleaner (without the parenthesis and +1). I've made these new ones mbetter discoverable throught the template comment.

Reply:
Yes, I understand, but in my opinion, developers shouldn't be surprised that indexes start at zero. Perhaps this can be noted in the comments of the `CurrentPageIndex` property.

Reply:
I'll remove those added `Display...` from the `State`. Comments already mention the indices are zero-based. I'll mention it in the template comments as well:
```
 /// <summary>
 /// Optionally supplies a template for rendering the pagination summary.
 /// The following values can be included:
 /// {your State parameter name}.CurrentPageIndex (zero-basedd, so +1 for the current page number)
 /// {your State parameter name}.LastPageIndex (zero-based, so +1 for the total number of pages)
 /// </summary>
```